framework/include/base/DependencyResolverInterface.h

```diff
@@ -23,6 +23,12 @@
 
 // MOOSE includes
 #include "DependencyResolver.h"
+#include "InputParameters.h"
+
+// Forward Declarations
+class DependencyResolverInterface;
+
+
```

Comment:
Looks like you left a little bit of stuff from your previous attempt. You probably don't need to touch this file at all. The remaining three changes are all whitespace.